src/components/Button/index.tsx

```diff
@@ -86,7 +84,7 @@ const StyledButton = styled.button<StyledButtonProps>`
     inset: -1px;
     background-color: transparent;
     border-radius: calc(${radius.md} + 1px);
-    box-shadow: 0 0 0 ${space[4]} ${color.actionFocus};
+    outline: ${space[4]} solid ${transparentize(color.action, opacity.statusFocusColor)};
```

Comment:
The reason for this change is because `color-mix` doesn't play nicely with css variables + `box-shadow` (yet)